### PR TITLE
Convert to yaml

### PR DESCRIPTION
**Pull request by LeandroStanger**

### CODE REVIEW

**Comment by TingPing** [comment]
https://github.com/flathub/flathub/wiki/YAML-Style-Guide

File: com.discordapp.Discord.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+- "--socket=pulseaudio"
+- "--share=network"
+- "--device=all"
+- "--talk-name=org.freedesktop.Notifications"
+- "--filesystem=xdg-videos:ro"
+- "--filesystem=xdg-pictures:ro"
+- "--filesystem=xdg-download"
+- "--filesystem=xdg-run/discord:create"
+- "--env=XDG_CURRENT_DESKTOP=Unity"
+- "--talk-name=org.kde.StatusNotifierWatcher"
```
Comment (sat3ll) [inline com.discordapp.Discord.yaml]: Quotes are unnecessary here.

File: com.discordapp.Discord.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    path: com.discordapp.Discord.appdata.xml
+  modules:
+  - name: libcxx
+    buildsystem: cmake-ninja
+    builddir: true
+    make-args:
+    - cxx
+    no-make-install: true
+    config-opts:
+    - "-DCMAKE_BUILD_TYPE=Release"
```
Comment (sat3ll) [inline com.discordapp.Discord.yaml]: Quotes are unnecessary here.

File: com.discordapp.Discord.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    cleanup:
+    - "/include"
+    - "/lib/*.a"
```
Comment (sat3ll) [inline com.discordapp.Discord.yaml]: Quotes are unnecessary here.

**Pull request closed without merging**